### PR TITLE
docs: add Nuxt server API route getSession example

### PR DESCRIPTION
**Pull request by TimvdEijnden**
Added an example to get the session on a server route. When implementing this I looked at the other integrations to see how it worked.

### CODE REVIEW

File: docs/content/docs/integrations/nuxt.mdx [hunk 1]
```diff
@@ -68,6 +68,27 @@ const session = authClient.useSession()
 </template>
 ```
 
+### Server Usage
+
+The `api` object exported from the auth instance contains all the actions that you can perform on the server. Every endpoint made inside Better Auth is a invocable as a function. Including plugins endpoints.
+
+**Example: Getting Session on a server API route**
+
+```tsx title="server/api/example.ts"
```
Comment (TimvdEijnden) [inline docs/content/docs/integrations/nuxt.mdx:77]: Thanks for the quick merge but `tsx` should actually be `ts`

Comment (TimvdEijnden) [inline docs/content/docs/integrations/nuxt.mdx:77]: Thanks for the quick merge but `tsx` should actually be `ts`, sorry 

@Bekacru

**Comment by poor-john** [comment]
Little fix here:
this code needs the "async" for being able to await for the promise.
`import { auth } from "~/lib/auth";
 
export default defineEventHandler( async (event) => {
    const session = await auth.api.getSession({
      headers: event.headers
    });
 
   if(session) {
     // access the session.session && session.user
   }
});`

**Comment by poor-john** [comment]
Little fix here:
this code needs the "async" for being able to await for the promise.
```typescript
import { auth } from "~/lib/auth";

export default defineEventHandler( async (event) => {
  const session = await auth.api.getSession({
    headers: event.headers
  });

  if(session) {
    // access the session.session && session.user
  }
});
```

**Pull request merged**